src/python/geoclaw/surge/storm.py

```diff
@@ -354,7 +354,10 @@ def read_atcf(self, path, verbose=False):
             #             self.time_offset = self.t[i]
 
             # Classification, note that this is not the category of the storm
-            self.classification[i] = data[10]
+            try:
+                self.classification[i] = data[10]
+            except IndexError:
+                self.classification[i] ''
```

Comment:
Missing an equal sign here.

Reply:
Thanks! My fix for that typo clearly didn't make it into my commit.